src/components/_profile.scss

```diff
@@ -0,0 +1,119 @@
+@use '../base.scss';
+
+.profile--dark {
+  background-color: base.$secondary-color;
+  color: base.$font-white;
+  line-height: 1.7em;
+  padding-bottom: 0.8rem;
+  padding-top: 0.8rem;
+}
+
+.profile__personal {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  padding-top: 0.8em;
+  padding-bottom: 0.8rem;
+}
```

Comment:
if you use `display:flex;` you can also use `flex-direction: row; ` (which is the default) or `flex-direction: column; `. Maybe this helps.

Reply:
Didn't work like I intended it to do, but looks even better now 😃